### PR TITLE
Docs cleanup

### PR DESCRIPTION
**Pull request by bocajnotnef**
Made sure docs could actually be produced
Cleaned up the toctree
Renamed CODE_OF_CONDUCT to fit convention
Adjusted how sphinx got screed's version
Removed unnecessary files

### CODE REVIEW

**Comment by bocajnotnef** [comment]
test this please

**Comment by bocajnotnef** [comment]
please test this

**Comment by bocajnotnef** [comment]
add to testlist

**Comment by bocajnotnef** [comment]
test this please

**Comment by bocajnotnef** [comment]
test this, please

**Comment by bocajnotnef** [comment]
jenkins, test this please

**Comment by bocajnotnef** [comment]
Resolves https://github.com/ged-lab/khmer/issues/800

**Comment by bocajnotnef** [comment]
@mr-c CR? (Also, I can't get jenkins to test this)

**Comment by bocajnotnef** [comment]
@mr-c glancing CR part II, please

**Comment by mr-c** [comment]
```
$ make doc
make: Nothing to be done for `doc'.
```

**Comment by bocajnotnef** [comment]
- [ ] harmonize khmer and screed doc conf files

File: conf.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
+# -*- coding: utf-8 -*-
```
Comment (mr-c) [inline conf.py]: This file goes in the doc directory, not the project root

**Comment by bocajnotnef** [comment]
TODO:

 - [ ] move conf.py
 - [ ] diff khmer/screed conf.pys, fix major discrepancies
 - [ ] CR & merge

@ctb

**Comment by bocajnotnef** [comment]
Relevant to https://github.com/ged-lab/khmer/issues/800

**Comment by bocajnotnef** [comment]
@mr-c CR please?

File: doc/conf.py [hunk 2]
```diff
@@ -41,16 +41,28 @@
 
 # General information about the project.
 project = u'screed'
-copyright = u'2012, Alex Nolley and Titus Brown'
+copyright = u'2015, Alex Nolley and Titus Brown'
```
Comment (mr-c) [inline doc/conf.py]: 2012-2015

Comment (ctb) [inline doc/conf.py]: On Tue, Feb 24, 2015 at 02:01:13PM -0800, Michael R. Crusoe wrote:
> > @@ -41,16 +41,28 @@
> >  
> >  # General information about the project.
> >  project = u'screed'
> > -copyright = u'2012, Alex Nolley and Titus Brown'
> > +copyright = u'2015, Alex Nolley and Titus Brown'
> 
> 2012-2015

copyright should be to Michigan State University, too!

Comment (ctb) [inline doc/conf.py]: I mean, instead of Alex and myself.  We're the authors but not the (c)
holders.

**Comment by bocajnotnef** [comment]
@mr-c Should be clean now

File: doc/conf.py [hunk 3]
```diff
@@ -41,16 +41,28 @@
...
 # built documents.
 #
-# The short X.Y version.
-version = '0.7'
+
 # The full version, including alpha/beta/rc tags.
-release = '0.7.1'
+
+sys.path.insert(0, '.')
+import versioneer
```
Comment (mr-c) [inline doc/conf.py]: Might be wise to use the same hack around the current versioneer import bug that `setup.py` uses: https://github.com/ged-lab/screed/blob/master/setup.py#L8

File: Makefile [hunk 4]
```diff
@@ -89,6 +90,14 @@ nosetests.xml: FORCE
 
 doxygen: doc/doxygen/html/index.html
 
+doc: build/sphinx/html/index.html
```
Comment (mr-c) [inline Makefile]: `make doc` fails. Add sphinx to line 10

**Comment by bocajnotnef** [comment]
@mr-c @ctb Should be clean now.

**Pull request merged**